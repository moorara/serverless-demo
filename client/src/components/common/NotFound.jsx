import React from 'react'

const NotFound = () => (
  <section className="hero is-danger is-medium">
    <div className="hero-body">
      <div className="container">
        <h1 className="title">Not Found</h1>
        <h2 className="subtitle">The page you requested does not exist!</h2>
      </div>
    </div>
  </section>
)

export default NotFound
