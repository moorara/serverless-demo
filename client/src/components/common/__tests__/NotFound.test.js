import React from 'react'
import renderer from 'react-test-renderer'

import NotFound from '../NotFound'

describe('NotFound', () => {
  test('renders react component', () => {
    const component = renderer.create(<NotFound />)
    const tree = component.toJSON()
    const instance = component.root

    expect(tree).toMatchSnapshot()
  })
})
