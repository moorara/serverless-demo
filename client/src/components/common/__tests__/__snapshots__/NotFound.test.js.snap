// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound renders react component 1`] = `
<section
  className="hero is-danger is-medium"
>
  <div
    className="hero-body"
  >
    <div
      className="container"
    >
      <h1
        className="title"
      >
        Not Found
      </h1>
      <h2
        className="subtitle"
      >
        The page you requested does not exist!
      </h2>
    </div>
  </div>
</section>
`;
